package.json

```diff
@@ -113,7 +113,7 @@
     "react-portal": "^3.0.0"
   },
   "peerDependencies": {
-    "moment": "2.10 - 2.14 || ~2.15.1 || ~2.16 || ~2.17",
+    "moment": "^2.18.1",
```

Comment:
to be a patch, this needs to be `2.10 - 2.14 || ~2.15.1 || ~2.16 || ~2.17 || ^2.18.1`